core/src/main/scala/scalaz/EitherT.scala

```diff
@@ -324,10 +324,6 @@ sealed abstract class EitherTInstances2 extends EitherTInstances3 {
 }
 
 sealed abstract class EitherTInstances1 extends EitherTInstances2 {
-  implicit def eitherTMonad[F[_], L](implicit F0: Monad[F]): Monad[EitherT[F, L, ?]] =
```

Comment:
This is unnecessary because same restriction of `eitherTMonadError`

https://github.com/scalaz/scalaz/blob/b9666d892ef287657156dd9b2c62bd3945c02817/core/src/main/scala/scalaz/EitherT.scala#L313